integration-tests/tests/index.test.ts

```diff
@@ -0,0 +1,122 @@
+import 'dotenv/config'
```

Comment:
These don't run in CI (because the v0.5 branch doesn't have CI set up). I have confirmed that they pass locally.

I've also manually tested `decryptBulkFallible` with lock contexts and locally for the happy (all data keys succeed) and unhappy (partial success) paths.